### PR TITLE
macchanger: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/macchanger.md [hunk 1]
```diff
@@ -11,10 +11,22 @@
 
 `macchanger --random {{interface}}`
 
+- Set interface to a random MAC, and pretend to be a [b]urned-[i]n-[a]ddress:
```
Comment (gutjuri) [inline pages/linux/macchanger.md:14]:
```suggestion
- Set an interface to a random MAC address, and pretend to be a [b]urned-[i]n-[a]ddress:
```

File: pages/linux/macchanger.md [hunk 2]
```diff
@@ -11,10 +11,22 @@
...
+
+`macchanger --random --bia {{interface}}`
+
+- Set interface to a random vendor MAC of same/any kind:
```
Comment (gutjuri) [inline pages/linux/macchanger.md:18]:
```suggestion
- Set an interface to a random vendor MAC address of same/any kind:
```

Also, please clarify: same kind with regards to what?

File: pages/linux/macchanger.md [hunk 3]
```diff
@@ -11,10 +11,22 @@
...
+
+`macchanger -{{a|A}} {{interface}}`
+
 - Set interface to a specific MAC:
```
Comment (gutjuri) [inline pages/linux/macchanger.md:22]:
```suggestion
- Set interface to a specific MAC address:
```

File: pages/linux/macchanger.md [hunk 4]
```diff
@@ -11,10 +11,22 @@
...
 
 `macchanger --mac {{XX:XX:XX:XX:XX:XX}} {{interface}}`
 
+- Print identification (first 3 pairs of MAC) of all known vendors:
```
Comment (gutjuri) [inline pages/linux/macchanger.md:26]:
```suggestion
- Print the identifications (the first three bytes of a MAC address) of all known vendors:
```

File: pages/linux/macchanger.md [hunk 5]
```diff
@@ -11,10 +11,22 @@
...
+
+`macchanger --list`
+
 - Reset interface to its permanent hardware MAC:
```
Comment (gutjuri) [inline pages/linux/macchanger.md:30]:
```suggestion
- Reset an interface to its permanent hardware MAC address:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions.

File: pages/linux/macchanger.md (hunk 2 above)
Comment (fazlearefin) [reply pages/linux/macchanger.md:18]: I will remove this example. I have tested it, and it the tool does not comply with the documentation. So this is a buggy feature.

**Review by gutjuri — APPROVED** [review]

**Pull request merged**